Week3/homework/1-step3.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
+  func();
 }
 
```

Comment:
Hi Karam, 
It is generally a good practice to put the call to func in a return statement, so the value is returned from your function.  In filter, map and reduce it is the return value that is important.
```
return func();
```
Otherwise, nice answer.